content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -519,6 +519,24 @@ section.
 
 {{% code_sample file="service/networking/tls-example-ingress.yaml" %}}
 
+{{< note >}}
+As an alternative to make TLS work on the default rule or when faced with specific 
+Subject Alternative Name requirements (e.g mandatory IPs instead of FQDN), user can update the 
+Ingress Controller to set as default certificate a custom Secret that satifies those requirements.
+
+If using Nginx Ingress:
+
+kubectl -n ingress-nginx edit deployment ingress-controller
+
+Edit the args section as follows:
+
+args:
+ - /nginx-ingress-controller
+ - '--publish-service=$(POD_NAMESPACE)/ingress-nginx-controller'
+ - '--default-ssl-certificate=default/testsecret-tls'
+ - ...
+{{< /note >}}
```

Comment:
This advice is specific to one ingress controller. We could add it as part of a blog article or a tutorial, but I don't think it's right for a page about the Ingress API as a concept.

Reply:
I would agree with that indeed